.github/workflows/pre-commit.yml

```diff
@@ -183,6 +183,15 @@ jobs:
           repository: StratusGrid/workflow-config
           ref: main
           path: workflow-config
+      - name: Terraform Automation get install token
+        id: get-token
+        uses: getsentry/action-github-app-token@38a3ce582e170ddfe8789f509597c6944f2292a9
```

Comment:
Can you reference the version tag in a comment and add a comment at the top of the file explaining why we're doing commit SHAs please, you can link to that reference you sent me.

Reply:
I pined to SHA with commented version.